.github/workflows/issues.yml

```diff
@@ -10,6 +10,13 @@ jobs:
   triage-labels:
     runs-on: ubuntu-latest
     steps:
+      - name: If no labels, add to scrum board
+        uses: actions/add-to-project@v0.3.0
+        if: ${{ join(github.event.issue.labels.*.name, ',') == '' }}
```

Comment:
Not sure about the workflow: Will this trigger if any time in the lifetime of the issue, there are no labels?
Should we check for `is:open`
Also, what if it is transferred? 

I think it is a good start.

Reply:
What if I change it to add to the board if the "triage-needed" label is added?